tracer/src/Datadog.Trace/Debugger/ExceptionAutoInstrumentation/ExceptionDebugging.cs

```diff
@@ -78,14 +78,14 @@ private static void InitSnapshotsSink()
             var discoveryService = tracer.TracerManager.DiscoveryService;
             var gitMetadataTagsProvider = tracer.TracerManager.GitMetadataTagsProvider;
 
-            var snapshotBatchUploadApi = AgentBatchUploadApi.Create(apiFactory, discoveryService, gitMetadataTagsProvider, false);
+            var snapshotBatchUploadApi = AgentBatchUploadApi.CreateSnapshotApi(apiFactory, discoveryService, gitMetadataTagsProvider, isDiagnostics: false);
```

Comment:
I think this is a typo now?
```suggestion
            var snapshotBatchUploadApi = AgentBatchUploadApi.CreateSnapshotApi(apiFactory, discoveryService, gitMetadataTagsProvider);
```

Reply:
Oh, right. Fixed here: 22c62454da2eeb3706030c043400ea9c7a81de1f